README.rst

```diff
@@ -11,7 +11,7 @@ Overview
 Currently PySlurm is under development to move from it's thin layer on top of
 the Slurm C API to an object orientated interface.
 
-This release is based on Slurm 19.05.
+This release is based on Slurm 20.05.
```

Comment:
This should be 20.02.

Reply:
Whoops, fixed it, don't know why I put .05 there